pkg/virtctl/vm/stop.go

```diff
@@ -65,19 +64,20 @@ func (o *Command) stopRun(args []string, cmd *cobra.Command) error {
 		dryRunOption = []string{metav1.DryRunAll}
 		fmt.Printf("Dry Run execution\n")
 	}
+	
+	gracePeriodChanged := cmd.Flags().Changed(gracePeriodArg)
 
-	if cmd.Flags().Changed(gracePeriodArg) && forceRestart == false {
-		return fmt.Errorf("Can not set gracePeriod without --force=true")
+	if gracePeriodChanged != forceRestart {
+		if !forceRestart {
+			return fmt.Errorf("Can not set gracePeriod without --force=true")
+		}
+		return fmt.Errorf("Can not force stop without gracePeriod")
```

Comment:
I do not think we need to be that verbose about the error, it could be just one combined message.
But this is a nit.

Reply:
I consolidated this into a single error message.